cmake/projects/qhull/hunter.cmake

```diff
@@ -12,11 +12,11 @@ hunter_add_version(
     PACKAGE_NAME
     qhull
     VERSION
-    7.2.0-p0
+    7.2.0-p1
```

Comment:
Do not modify old entry, add new instead:
* https://docs.hunter.sh/en/latest/creating-new/update.html#update-package